src/main/scala/cromwell/engine/backend/local/LocalEngineFunctions.scala

```diff
@@ -15,7 +15,9 @@ class LocalEngineFunctions(executionContext: TaskExecutionContext) extends Engin
   override protected def read_int(params: Seq[Try[WdlValue]]): Try[WdlInteger] = {
```

Comment:
They are running in a `map` over the `Try[WdlValue]` returned by `assertSingleArgument`.  A forcomp would probably be more clear.